### ISSUE TITLE
Private browsing windows issue with forced one-window mode

### ISSUE THREAD

**Issue by IamProRu** [onemen/TabMixPlus#96]
If I-m using forced one window mode in TabMixPlus I can't open private browsing window with Firefox main menu, Firefox tollbar menu and keyboard shortcut (Ctrl+Shift+P) - it only opens new tab in current window. Open it private window in link's toolbar on web page also opens link in new tab is same window, not in private browsing window.
But it works if add new private window button to toolbar and hit it. Also I can open private window when visits about:privatebrowsing page in and hit Open private window button here.
TabMixPlus 1.0.0-pre.3
Firefox 93 through 94.0.1

**Comment by onemen** [comment]
Attached a file with your Tab mix preference

**Comment by IamProRu** [comment]
[TMPpref.txt](https://github.com/onemen/TabMixPlus/files/7503261/TMPpref.txt)

**Comment by IamProRu** [comment]
I have migrated from Firefox 56 with latest TabMixPlus version was available at addons.mozilla.com to Firefox 93 by one major version at a time on mu desktop about three weeks ago and when installed new TabMixPlus 1.0.0-pre.3.
Attaching an old TMPpref from Firefox 56 [TMPpref-56.txt](https://github.com/onemen/TabMixPlus/files/7503310/TMPpref-56.txt), just in case.
Also I had an error after migrating with two keyboard shortcut actions assigned with one hotkeys combination in TMP. I've deleted them both and when assigned only one of them.
Then I did the same on my laptop with almost the same options in TMP. Firefox browser settings Sync was disabled and all others Sync elements was enabled. Where was no shortcut collision in TMP preferences
